pkg/proxy/winuserspace/proxier.go

```diff
@@ -99,26 +97,6 @@ type Proxier struct {
 // assert Proxier is a proxy.Provider
 var _ proxy.Provider = &Proxier{}
 
-// A key for the portMap.  The ip has to be a string because slices can't be map
```

Comment:
Are these pieces removed because they are not used?

Reply:
Yes, from the commit message:
```
pkg/proxy/winuserspace/proxier.go:88:2: field portMapMutex is unused (U1000)
pkg/proxy/winuserspace/proxier.go:112:2: field owner is unused (U1000)
pkg/proxy/winuserspace/proxier.go:113:2: field socket is unused (U1000)
```